src/main/java/com/example/MultipleChoiceQuestion.java

```diff
@@ -9,22 +9,23 @@
 
 @Entity
 public class MultipleChoiceQuestion extends SurveyQuestion {
-    private String description; // question desc
     private int numChoices; // number of MC options user can select from
     private int selectedOption = -1; // the option the user selected
 
     @ElementCollection
     private List<String> options = new ArrayList<>();
 
 
-    public MultipleChoiceQuestion(String description, int numChoices, ArrayList<String> options) {
-        this.description = description;
+    public MultipleChoiceQuestion(int numChoices, ArrayList<String> options) {
+        super();
+        this.questionType = QuestionType.valueOf("MULTIPLE_CHOICE");
```

Comment:
Yup, we have a file named QuestionType.java for that purpose.

Reply:
I think it should be formatted as:
QuestionType.MULTIPLE_CHOICE
rather than:
QuestionType.valueOf("MULTIPLE_CHOICE");

Intellij will highlight the first one as an error if you miss spell it, but not for the second way

Reply:
Good point, added!